tests/monotouch-test/Vision/VNUtilsTests.cs

```diff
@@ -0,0 +1,60 @@
+//
+// Unit tests for VNUtils
+//
+// Authors:
+//	Rachel Kang <rachelkang@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+#if !__WATCHOS__
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using OpenTK;
+
+using CoreGraphics;
+using Foundation;
+using Vision;
+
+namespace MonoTouchFixtures.Vision {
+
+    [TestFixture]
+    [Preserve (AllMembers = true)]
+    public class VNUtilsTests {
+
+        [SetUp]
+        public void Setup () => TestRuntime.AssertXcodeVersion (13, 0);
+
+        [Test]
+        public void GetImagePointTest ()
+        {
+            var imagePoint = VNUtils.GetImagePoint (new CGPoint(0, 0), 1, 1, new CGRect(0, 0, 1, 1));
+            Assert.NotNull (imagePoint, $"VNImagePointForNormalizedPointUsingRegionOfInterest is not null");
```

Comment:
`CGPoint` is a struct, so a NotNull check will always succeed.

Maybe check that it's not an empty point instead?

```csharp
Assert.That (imagePoint , Is.Not.EqualTo (CGPoint.Empty), "...");
```

(this might have to be slightly modified if NUnit isn't able to figure out how to compare CGPoints, I'm not quite sure about the details in this case)